### PR TITLE
Support 64-bit images for relocation.

### PR DESCRIPTION
**Pull request by floitsch**

### CODE REVIEW

File: tests/image/relocation-test.toit [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    pipe.run-program toitrun "-w" snap-file toit-file
+
+    pipe.run-program [toitrun, snapshot-to-image, "--format", "binary", "-m32", "-o", img32-file, snap-file]
+    pipe.run-program [toitrun, snapshot-to-image, "--format", "binary", "-m64", "-o", img64-file, snap-file]
+
+    snapshot := file.read-content snap-file
+    snapshot-bundle := SnapshotBundle "snapshot" snapshot
+    snapshot-uuid ::= snapshot-bundle.uuid
+
+    word-size := 4
```
Comment (kasperl) [inline tests/image/relocation-test.toit:45]: Maybe pull this out into a parameterized test helper?

File: tests/package.yaml [hunk 2]
```diff
@@ -17,6 +17,9 @@ dependencies:
   host:
     url: github.com/toitlang/pkg-host
     version: ^1.15.1
+  partition-table:
```
Comment (kasperl) [inline tests/package.yaml:20]: Do we need this?

**Review by kasperl — APPROVED** [review]
Looks good to me.

File: tests/image/relocation-test.toit (hunk 1 above)
Comment (floitsch) [reply tests/image/relocation-test.toit]: made the test-help almost the entire main function.
done.

File: tests/package.yaml (hunk 2 above)
Comment (floitsch) [reply tests/package.yaml:20]: yes. We are including the `firmware` library, which needs the partition-table package.

**Pull request merged**